packages/components/src/internal/components/domainproperties/DomainForm.spec.tsx

```diff
@@ -684,4 +684,59 @@ describe('DomainForm', () => {
         expect(form).toMatchSnapshot();
         form.unmount();
     });
+
+    test('using allowImportExport', () => {
+        const domain = DomainDesign.create({});
+
+        const form = mount(
+            <DomainForm
+                domain={domain}
+                onChange={jest.fn()}
+                allowImportExport={true}
+            />
+        );
+
+        expect(form.find('.domain-form-manual-section').length).toEqual(1);
+        expect(form.find('.domain-form-manual-section').length).toEqual(1);
+        expect(form.find('.file-form-formats').text()).toContain('.json');
+
+        expect(form).toMatchSnapshot();
```

Comment:
These DomainForm related snapshot tests create such insanely long test artifacts that they are often not very useful or easy to maintain. I suggest avoiding adding new ones if possible. Any chance these test cases for the enzyme checks could cover the various cases? 

For example in this case, you are checking the section class exists once and the format text contains .json. That is perfect. If we can just add some "negative checks" as well for the things that would be there in the opposite case (i.e. allowImportExport = false), then I think this would cover the scenario without needing a snapshot test.

Reply:
That makes more sense for sure! Updated, and 'using allowImportExport' removed.